src/oxr/index.ts

```diff
@@ -0,0 +1,60 @@
+export type OXEClient = ReturnType<typeof createOpenExchangeClient>
+
+export type OXEInternationalCurrencies = "USD" | "EUR" | "GBP" | "JPY" | "BRL";
+
+export interface OXEQueryParams {
+  'app_id': string,
+  readonly [key: string]: string | number | boolean;
+}
+
+export interface OXEDollarRates {
+  rates: Record<string, number>;
+  base: string;
+  timestamp: number;
+  license: string;
+  disclaimer: string;
+}
```

Comment:
OXE or OXR? 👀

Reply:
also. why not move this types to the types file?

Reply:
good call, will fix for OXR